lib/ruby_indexer/lib/ruby_indexer/index.rb

```diff
@@ -5,6 +5,9 @@ module RubyIndexer
   class Index
     extend T::Sig
 
+    # The minimum Jaro-Winkler similarity score for an entry to be considered a match for a given fuzzy search query
+    ENTRY_SIMILARITY_THRESHOLD = 0.7
```

Comment:
For my own curiosity, is it a standard threshold for other fuzzy matchers? I wonder what the typescript extension uses.

Reply:
For future reference, the [implementation used by VSCode](https://github.com/microsoft/vscode/blob/515c6b204fefa52688c8ff92757ff69d0917a36f/src/vs/base/common/fuzzyScorer.ts) is interesting. It looks similar to Jaro-Winkler with custom heuristic for special chars. May be useful for fuzzing methods with `A#foo` vs. `A::foo`.

Reply:
I spent a long time trying to find it in the repo but clearly your search skills bested mine 😂. Thanks for sharing, this is useful to know.

We ended up deciding to go with the implementation from [DidYouMean](https://github.com/Shopify/ruby-lsp/pull/906#issuecomment-1684104456) to avoid maintaining it, but I do believe that some custom tuning could potentially provide better results.